settings.gradle.kts

```diff
@@ -0,0 +1,14 @@
+// TODO(egorand): Remove once gradle-maven-publish-plugin:0.8.0 is published
```

Comment:
There still seems to be an issue with the dokka documentation. I haven't forgotten about this

Reply:
Yeah, too bad `plugins {}` makes it this hard to consume plugins from outside of the plugin portal. Not a big fan.